deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/wazuh_deployment/wazuh_local_package.py

```diff
@@ -9,6 +9,24 @@
 
 
 class WazuhLocalPackage(WazuhPackage):
+    """Install Wazuh from a local existent package
+
+    Args:
+        wazuh_target (string): Type of the Wazuh instance desired (agent or manager).
+        installation_files_path (string): Path where is located the Wazuh instalation files.
+        local_package_path (string): Path where the local package is located.
+        qa_ctl_configuration (QACTLConfiguration): QACTL configuration.
+        version (string): The version of Wazuh. Parameter set by default to 'None'.
+        system (string): System of the Wazuh installation files. Parameter set by default to 'None'.
+
+    Attributes:
+        local_package_path (string): Path where the local package is located.
+        package_name (string): name of the Wazuh package.
+        installation_files_path (string): Path where the Wazuh installation files are located.
+        local_package_path (string): Path where the local package is located.
+        qa_ctl_configuration (QACTLConfiguration): QACTL configuration.
+
+    """
```

Comment:
```suggestion
    """
```

Reply:
The new modification can be reviewed in the commit:[bba01140837453f908a9dafefa89308ee9a5103e](https://github.com/wazuh/wazuh-qa/commit/bba01140837453f908a9dafefa89308ee9a5103e)